packages/eslint-plugin/src/configs/react.js

```diff
@@ -264,6 +282,8 @@ const config = {
     'import/named': 'off',
     'import/namespace': 'off',
     'import/no-named-as-default-member': 'off',
+
+    'jsdoc/check-tag-names': ['error', { definedTags: ['jest-environment'] }],
```

Comment:
this is a topic to be discussed at vbuild sync, but I'd propose to use severity `warn` - lint violation is not inflicting any critical issues to runtime code - thus should not break build etc.

In general our DX is a big red squiggles ocean (for non critical violations) which is from my experience (based on feedback of big teams) very distracting.

Reply:
In my experience, warnings never get fixed and just clutter up the build output. If you have warnings checked into master, then people learn to ignore warnings (and new ones get buried in the sea of warnings that other people haven't fixed). I'd say it's not worth adding this at all unless it's an error.

Reply:
What @behowell said is generally what I've also said when we discussed this in the past. It seems like a fundamental philosophical difference about the purpose of linting, which I'm not sure how to resolve.

Reply:
> In my experience, warnings never get fixed and just clutter up the build output. If you have warnings checked into master, then people learn to ignore warnings (and new ones get buried in the sea of warnings that other people haven't fixed). I'd say it's not worth adding this at all unless it's an error.

everyone has different experience for sure :) IF you have solid CI DX (integrated into review - eg github action) and rule sets (to fail pipeline if there is a threshold reached of lint warnings), it is very nicely doable (implemented this  for 100 dev org and it worked very nicely)